lesson_01_hw_01/1.py

```diff
@@ -15,7 +15,7 @@ def deco(*args, **kwargs):
             cache_key = (args, tuple(kwargs.items()))
 
             if cache_key in deco._cache:
-                # збільшуємо лічильник на одиницю
```

Comment:
line 26.
we should start from 1, {'counter': 1, 'content': result}
line 30.
if len(deco._cache) > max_limit:
remove prints